pkg/plugins/backendplugin/backendplugin.go

```diff
@@ -2,8 +2,9 @@
 package backendplugin
 
 import (
+	"github.com/grafana/grafana/pkg/infra/tracing"
 	"github.com/grafana/grafana/pkg/plugins/log"
 )
 
 // PluginFactoryFunc is a function type for creating a Plugin.
-type PluginFactoryFunc func(pluginID string, logger log.Logger, env func() []string) (Plugin, error)
+type PluginFactoryFunc func(pluginID string, logger log.Logger, tracer tracing.Tracer, env func() []string) (Plugin, error)
```

Comment:
We have been conscious of not depending on grafana packages from `pkg/plugins` where possible - so in that spirit, would it be possible to replace `tracing.Tracer` with `trace.Tracer` from `"go.opentelemetry.io/otel/trace"`?